### DISCUSSION TITLE
non particle-specific DST info

### DISCUSSION THREAD

**Discussion by naidoo88** [dglazier/chanser#26] [User problems] [answered]
Hello, 

I'm looking to access non-particle specific, I suppose event-wide, DST info, inside the "main" Pi0.cpp final state file. 

Specifically I want to do something analogous to:
`helicity = _c12->event()->getHelicity();`
to access the helicty of each event.

Apologies if I'm missing an example of this somewhere! 

Cheers,
Paul

**Comment by dglazier** [comment, marked as answer]
This is held in the EventInfo so you should access it with
GetEventInfo()->_BeamHel

Other EventInfo are set in HipoData,

   ```
_eventInfo._TrigBit=_c12->runconfig()->getTrigger();
    _eventInfo._StartTime=_c12->event()->getStartTime();
    _eventInfo._FTBStartTime=_c12->event()->getFTBStartTime();
    _eventInfo._RFTime=_c12->event()->getRFTime();
    _eventInfo._BeamHel=_c12->event()->getHelicity();
    _eventInfo._NEvent=_c12->runconfig()->getEvent(); 
```

**Reply by naidoo88** [reply]
Great, cheers Derek!
